### ISSUE TITLE
[General Issue] Intel MKL DFTI ERROR: Incorrect status

### ISSUE THREAD

**Issue by jyh3690** [PaddlePaddle/PaddleSeg#2109]
1.PaddleSeg version :v2.2
2. PaddlePaddle version:paddlepaddle-gpu   2.2.2.post101
3:linux
4:python3.8
5:cuda10.1 cudnn7.6
6:但我运行matting中的python predict.py --config configs/modnet/modnet-resnet50_vd.yml --model_path weights/modnet-resnet50_vd.pdparams  --image_path  data/fg/时，报如下错误：Intel MKL DFTI ERROR: Incorrect status

这个我真的解决不了，能帮我一下嘛？谢谢

**Comment by haoyuying** [comment]
可以详细粘贴下报错栈信息吗

**Comment by jyh3690** [comment]
W0513 09:35:16.941200 84063 init.cc:143] Compiled with WITH_GPU, but no GPU found in runtime.
/home/spring/anaconda3/envs/paddleseg/lib/python3.8/site-packages/paddle/fluid/framework.py:311: UserWarning: You are using GPU version Paddle, but your CUDA device is not set properly. CPU device will be used by default.
  warnings.warn(
/home/spring/jyh/PaddleSeg/paddleseg/models/losses/decoupledsegnet_relax_boundary_loss.py:19: DeprecationWarning: Please use `shift` from the `scipy.ndimage` namespace, the `scipy.ndimage.interpolation` namespace is deprecated.
  from scipy.ndimage.interpolation import shift
/home/spring/jyh/PaddleSeg/paddleseg/transforms/functional.py:18: DeprecationWarning: Please use `distance_transform_edt` from the `scipy.ndimage` namespace, the `scipy.ndimage.morphology` namespace is deprecated.
  from scipy.ndimage.morphology import distance_transform_edt
/home/spring/jyh/PaddleSeg/Matting/metric.py:23: DeprecationWarning: Please use `convolve` from the `scipy.ndimage` namespace, the `scipy.ndimage.filters` namespace is deprecated.
  from scipy.ndimage.filters import convolve
OMP: Info #276: omp_set_nested routine deprecated, please use omp_set_max_active_levels instead.
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: ResNet18_vd exists already! It is now updated to <function ResNet18_vd at 0x7fab035d43a0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: ResNet50_vd exists already! It is now updated to <function ResNet50_vd at 0x7fab035d4ca0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: ResNet101_vd exists already! It is now updated to <function ResNet101_vd at 0x7fab035d4dc0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: MobileNetV2 exists already! It is now updated to <function MobileNetV2 at 0x7fab035d50d0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W18_Small_V1 exists already! It is now updated to <function HRNet_W18_Small_V1 at 0x7fab035e6160> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W18_Small_V2 exists already! It is now updated to <function HRNet_W18_Small_V2 at 0x7fab035e6dc0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W18 exists already! It is now updated to <function HRNet_W18 at 0x7fab035e6e50> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W30 exists already! It is now updated to <function HRNet_W30 at 0x7fab035e6f70> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W32 exists already! It is now updated to <function HRNet_W32 at 0x7fab035ea0d0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W40 exists already! It is now updated to <function HRNet_W40 at 0x7fab035ea1f0> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W44 exists already! It is now updated to <function HRNet_W44 at 0x7fab035ea310> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W48 exists already! It is now updated to <function HRNet_W48 at 0x7fab035ea430> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W60 exists already! It is now updated to <function HRNet_W60 at 0x7fab035ea550> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: HRNet_W64 exists already! It is now updated to <function HRNet_W64 at 0x7fab035ea670> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: Compose exists already! It is now updated to <class 'transforms.Compose'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: Resize exists already! It is now updated to <class 'transforms.Resize'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: ResizeByLong exists already! It is now updated to <class 'transforms.ResizeByLong'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: ResizeByShort exists already! It is now updated to <class 'transforms.ResizeByShort'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: Normalize exists already! It is now updated to <class 'transforms.Normalize'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: LimitLong exists already! It is now updated to <class 'transforms.LimitLong'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: RandomHorizontalFlip exists already! It is now updated to <class 'transforms.RandomHorizontalFlip'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: RandomBlur exists already! It is now updated to <class 'transforms.RandomBlur'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: RandomDistort exists already! It is now updated to <class 'transforms.RandomDistort'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: Padding exists already! It is now updated to <class 'transforms.Padding'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/manager.py:113: UserWarning: RandomNoise exists already! It is now updated to <class 'transforms.RandomNoise'> !!!
  warnings.warn("{} exists already! It is now updated to {} !!!".
v: 7605
2022-05-13 09:35:17 [INFO]
---------------Config Information---------------
batch_size: 16
iters: 100000
lr_scheduler:
  boundaries:
  - 40000
  - 80000
  type: PiecewiseDecay
  values:
  - 0.02
  - 0.002
  - 0.0002
model:
  backbone:
    pretrained: https://bj.bcebos.com/paddleseg/dygraph/resnet50_vd_ssld_v2.tar.gz
    type: ResNet50_vd
  pretrained: null
  type: MODNet
optimizer:
  momentum: 0.9
  type: sgd
  weight_decay: 4.0e-05
train_dataset:
  dataset_root: data/PPM-100
  mode: train
  train_file: train.txt
  transforms:
  - type: LoadImages
  - crop_size:
    - 512
    - 512
    type: RandomCrop
  - type: RandomDistort
  - type: RandomBlur
  - type: RandomHorizontalFlip
  - type: Normalize
  type: MattingDataset
val_dataset:
  dataset_root: data/PPM-100
  get_trimap: false
  mode: val
  transforms:
  - type: LoadImages
  - short_size: 512
    type: ResizeByShort
  - mult_int: 32
    type: ResizeToIntMult
  - type: Normalize
  type: MattingDataset
  val_file: val.txt
------------------------------------------------
/home/spring/jyh/PaddleSeg/paddleseg/cvlibs/config.py:332: UserWarning: `dataset_root` is not found. Is it correct?
  warnings.warn("`dataset_root` is not found. Is it correct?")
2022-05-13 09:35:18 [INFO]      Loading pretrained model from https://bj.bcebos.com/paddleseg/dygraph/resnet50_vd_ssld_v2.tar.gz
2022-05-13 09:35:19 [INFO]      There are 275/275 variables loaded into ResNet_vd.
2022-05-13 09:35:21 [INFO]      Number of predict images = 2
2022-05-13 09:35:21 [INFO]      Loading pretrained model from weights/modnet-resnet50_vd.pdparams
2022-05-13 09:35:24 [INFO]      There are 420/420 variables loaded into MODNet.
2022-05-13 09:35:24 [INFO]      Start to predict...
Intel MKL DFTI ERROR: Incorrect status
以上是我的输出信息，非常感谢！
